File: ticketProject7-CrossPlatform/TS-crossp/networking/request/AddBuyerRequest.cs
using System;
namespace networking.protocol
{

	using BuyerDTO = networking.dto.BuyerDTO;

	[Serializable]
	public class AddBuyerRequest : Request
	{
		private BuyerDTO buyer;

		public AddBuyerRequest(BuyerDTO buyer)
		{
			this.buyer = buyer;
		}

		public virtual BuyerDTO Buyer
		{
			get
			{
				return buyer;
			}
		}
	}

}

File: ticketProject6-RMI/TicketSystem/TicketNetworking/src/main/java/network/objectprotocol/AddBuyerRequest.java
package network.objectprotocol;

import network.dto.BuyerDTO;

public class AddBuyerRequest implements Request{
    private BuyerDTO buyerDTO;

    public AddBuyerRequest(BuyerDTO buyerDTO) {
        this.buyerDTO = buyerDTO;
    }

    public BuyerDTO getBuyer() {
        return buyerDTO;
    }
}

